js/theme.js

```diff
@@ -97,6 +105,31 @@ function ThemeNav () {
             .on('click', "[data-toggle='rst-current-version']", function() {
                 $("[data-toggle='rst-versions']").toggleClass("shift-up");
             })
+            .on('touchstart', ".wy-body-for-nav", function(event) {
+                if (event.touches.length === 1) {
+                    self.sidebarSwipe.startX = event.touches.item(0).clientX;
+                    self.sidebarSwipe.startTime = new Date().getTime()
+                } else {
+                    self.sidebarSwipe.startX = null;
+                }
+            })
+            .on('touchend', ".wy-body-for-nav", function(event) {
+                if (self.sidebarSwipe.startX) {
```

Comment:
```suggestion
                if (self.sidebarSwipe.startX === null) {
```
because startX can have a value of `0`

Reply:
Done.
Thanks

Reply:
I just tested it locally, and it is not working if the suggesstion is applied.

Reply:
Whoops, should be `!== null`